lib/src/files.rs

```diff
@@ -225,52 +226,19 @@ pub fn merge(removes: &[&[u8]], adds: &[&[u8]]) -> MergeResult {
                 resolved_hunk.extend(content);
             }
             DiffHunk::Different(parts) => {
-                let mut removed_parts = parts[..num_removes].to_vec();
```

Comment:
[No action needed] So nice to see this code simplified this much :) 👍

Reply:
Thanks. I Agree :)